WordPress/Classes/ViewRelated/Views/LoadingStatusView.swift

```diff
@@ -15,7 +15,7 @@ class LoadingStatusView: UIView {
 
     private lazy var titleLabel: UILabel = {
         let label = UILabel()
-        label.textColor = .white
+        label.textColor = .appBarText
```

Comment:
@ceyhun `Generating Preview` was not visible in light mode, Please correct me if we are not using the right color.

Reply:
Good catch! Looks good to me 👍